pom.xml

```diff
@@ -83,6 +83,9 @@
 	</modules>
 
 	<repositories>
+		<!-- Start concerned by comment FUSETOOLS-2547 -->
```

Comment:
I think concerned by is not the right terminology here. I'd rather put here something like "Start of repository definitions which cause a warning in early build (see FUSETOOLS-2547)"